test/e2e/suite/issuers/acme/certificate/webhook.go

```diff
@@ -179,19 +180,17 @@ var _ = framework.CertManagerDescribe("ACME webhook DNS provider", func() {
 
 			pollErr = wait.PollImmediate(500*time.Millisecond, time.Second*30,
 				func() (bool, error) {
-					l, err := f.CertManagerClientSet.CertmanagerV1alpha1().Challenges(f.Namespace.Name).List(metav1.ListOptions{
-						LabelSelector: "acme.cert-manager.io/order-name=" + order.Name,
```

Comment:
This change is required because we no longer set this label with the new implementation :tada: